ajax/libs/slippry/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "slippry",
+  "filename": "slippry.min.js",
+  "version": "1.4.0",
+  "description": "Responsive slider plugin for jQuery. Intrinsic trick to enable responsive resizing without js.",
+  "keywords": [
+    "jQuery",
+    "Responsive",
+    "slider",
+    "plugin",
+    "PIXELS"
+  ],
+  "author": "Lukas Jakob Hafner <luki@booncon.com>",
+  "homepage": "http://slippry.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/booncon/slippry.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/booncon/slippry.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "*.+(css|js|map)"
```

Comment:
@maruilian11 minify tool is https://github.com/PeterDaveHello/web-minify-helper.
I don't get you about source map url?

Could you please help review it again, thank you.